packages/cardano-services/src/StakePool/TypeormStakePoolProvider/mappers.ts

```diff
@@ -147,5 +147,15 @@ export const mapStakePoolsResult = (rawResult: PoolModel[]): Paginated<Cardano.S
   };
 };
 
-export const mapPoolStats = (rawResult: PoolStatsModel[]): PoolStats =>
-  rawResult.reduce((acc, curr) => (acc = { ...acc, [curr.status]: Number(curr.count) }), {} as PoolStats);
+export const mapPoolStats = (rawResult: PoolStatsModel[]): PoolStats => {
```

Comment:
This is tested in https://github.com/input-output-hk/cardano-js-sdk/pull/905/commits/720f90c1e2c5f23ecfe90241f66f4cd6c34ee24c as re-generated db doesn't have any 'retiring' pools